source/postprocess/visualization/material_properties.cc

```diff
@@ -275,16 +275,17 @@ namespace aspect
       ASPECT_REGISTER_VISUALIZATION_POSTPROCESSOR(MaterialProperties,
                                                   "material properties",
                                                   "A visualization output object that generates output "
-                                                  "for the material properties given by the material model."
-                                                  "There are a number of other visualization postprocessors "
-                                                  "that offer to write individual material properties. However, "
-                                                  "they all individually have to evaluate the material model. "
-                                                  "This is inefficient if one wants to output more than just "
-                                                  "one or two of the fields provided by the material model. "
+                                                  "for the material properties given by the material model. "
+                                                  "It is inefficient to output more than just "
+                                                  "one or two of the fields provided by the material model "
+                                                  "when the material model is evaluated again for each field. "
```

Comment:
Maybe we can also remove this part? There is no point in mentioning the inefficiency, if we cannot go the inefficient route anyway.